### PR TITLE
Fetching Notes from database to dashboard page

### PR DESCRIPTION
**Pull request by dsathe**
Fetching Notes from database to dashboard page
Fetched notes in dashboard.
Currently fetching first 70 characters of content and pasting

To-do related to above task
1) Remove tags from string before showing.
2) Creating Masonry layout using CSS.

### CODE REVIEW

**Comment by Parth18Shah** [comment]
I am unable to open the notes page by clicking on the card. Could you please correct it?

File: src/app/dashboard/page.tsx [hunk 1]
```diff
@@ -5,15 +5,31 @@ import React from 'react';
...
+    if (!userId) {
+        return new NextResponse('unauthorised', { status: 401 });
+    }
+    const notes: Note[] = await prisma.note.findMany({
+        where: {
+            userId: userId,
+        }
+    });
+    console.log("Notes are " + notes.length);
+    console.log(notes);
```
Comment (Parth18Shah) [inline src/app/dashboard/page.tsx]: Please remember to remove such log statement once you have verified that the data is fetched correctly.

**Comment by dsathe** [comment]
Added the following changes:- 
1) Changed href to notebooks
2) Made changes to UI to address text-overflow
3) Removed unnecessary messages

**Review by Parth18Shah — COMMENTED** [review]
Seems to be good. Merging your changes into Main.

File: src/app/dashboard/page.tsx (hunk 1 above)
Comment (Parth18Shah) [reply src/app/dashboard/page.tsx]: Issue solved.

**Review by Parth18Shah — APPROVED** [review]
Noted your changes.

**Pull request merged**